[2024-04-23T09:00:03.455+0900] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: usd_krw_prediction_v1.prepare_data scheduled__2024-04-22T06:00:00+00:00 [queued]>
[2024-04-23T09:00:03.459+0900] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: usd_krw_prediction_v1.prepare_data scheduled__2024-04-22T06:00:00+00:00 [queued]>
[2024-04-23T09:00:03.459+0900] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-04-23T09:00:03.467+0900] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): prepare_data> on 2024-04-22 06:00:00+00:00
[2024-04-23T09:00:03.470+0900] {standard_task_runner.py:57} INFO - Started process 23024 to run task
[2024-04-23T09:00:03.476+0900] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'usd_krw_prediction_v1', 'prepare_data', 'scheduled__2024-04-22T06:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/var/folders/m6/3b3l583d1z9f5sjd_k30vc7h0000gp/T/tmpry7_hx1e']
[2024-04-23T09:00:03.479+0900] {standard_task_runner.py:85} INFO - Job 29: Subtask prepare_data
[2024-04-23T09:00:03.621+0900] {task_command.py:410} INFO - Running <TaskInstance: usd_krw_prediction_v1.prepare_data scheduled__2024-04-22T06:00:00+00:00 [running]> on host lm-045768-00.local
[2024-04-23T09:00:03.672+0900] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='usd_krw_prediction_v1' AIRFLOW_CTX_TASK_ID='prepare_data' AIRFLOW_CTX_EXECUTION_DATE='2024-04-22T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-22T06:00:00+00:00'
[2024-04-23T09:00:03.674+0900] {subprocess.py:63} INFO - Tmp dir root location: /var/folders/m6/3b3l583d1z9f5sjd_k30vc7h0000gp/T
[2024-04-23T09:00:03.676+0900] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '/Users/boycho/Project/exchange/bat/prepare_data.sh ']
[2024-04-23T09:00:03.682+0900] {subprocess.py:86} INFO - Output:
[2024-04-23T09:00:03.736+0900] {subprocess.py:93} INFO - python /Users/boycho/Project/exchange/bat/../exchange/prepare_data.py --dt 2024-04-23 --naver_keyword_csv /Users/boycho/Project/exchange/bat/../data/naver_keyword_data.csv --kr_oil_hist_csv /Users/boycho/Project/exchange/bat/../data/kr_oil_historical_data.csv --us_10y_hist_csv /Users/boycho/Project/exchange/bat/../data/us_10year_bond_yield_historical_data.csv --us_1y_hist_csv /Users/boycho/Project/exchange/bat/../data/us_1year_bond_yield_historical_data.csv --us_bb_hist_csv /Users/boycho/Project/exchange/bat/../data/us_bb_high_yield_historical_data.csv --us_bb_tmp_csv /Users/boycho/Project/exchange/bat/../data/us_bb_tmp_data.csv --log_dir /Users/boycho/Project/exchange/bat/../log
[2024-04-23T09:02:05.929+0900] {subprocess.py:93} INFO - 2024-04-23 09:02:05,929 [ERROR] download_bb_junkbond_to_csv : Error tokenizing data. C error: Expected 2 fields in line 496, saw 4
[2024-04-23T09:02:05.931+0900] {subprocess.py:93} INFO -  (crawling.py:125)
[2024-04-23T09:02:06.773+0900] {subprocess.py:93} INFO - 2024-04-23 09:02:06,772 [INFO] {'ok': True, 'channel': 'C06UA137LSW', 'ts': '1713830526.431179', 'message': {'user': 'U06UZ3A4XC0', 'type': 'message', 'ts': '1713830526.431179', 'bot_id': 'B06UA9HHRB4', 'app_id': 'A06UZ2ZDG8G', 'text': 'Error tokenizing data. C error: Expected 2 fields in line 496, saw 4\n', 'team': 'T06U73ALH53', 'bot_profile': {'id': 'B06UA9HHRB4', 'app_id': 'A06UZ2ZDG8G', 'name': 'PYTHON Messengers', 'icons': {'image_36': 'https://a.slack-edge.com/80588/img/plugins/app/bot_36.png', 'image_48': 'https://a.slack-edge.com/80588/img/plugins/app/bot_48.png', 'image_72': 'https://a.slack-edge.com/80588/img/plugins/app/service_72.png'}, 'deleted': False, 'updated': 1713173324, 'team_id': 'T06U73ALH53'}, 'blocks': [{'type': 'rich_text', 'block_id': 'sA=0O', 'elements': [{'type': 'rich_text_section', 'elements': [{'type': 'text', 'text': 'Error tokenizing data. C error: Expected 2 fields in line 496, saw 4'}]}]}]}} (msg_alerts.py:21)
[2024-04-23T09:03:32.062+0900] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-23T09:03:32.096+0900] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=usd_krw_prediction_v1, task_id=prepare_data, execution_date=20240422T060000, start_date=20240423T000003, end_date=20240423T000332
[2024-04-23T09:03:32.131+0900] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-04-23T09:03:32.154+0900] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
